apps/mark-scan/backend/src/app.test.ts

```diff
@@ -487,3 +482,88 @@ test('setPatDeviceIsCalibrated', async () => {
   await apiClient.setPatDeviceIsCalibrated();
   await waitForStatus('not_accepting_paper');
 });
+
+function sortVotes(votes: VotesDict): VotesDict {
+  return mapObject(votes, (vote) => {
+    assert(vote);
+
+    if (typeof vote[0] === 'object') {
+      return (vote as CandidateVote)
+        .slice()
+        .sort((a, b) => a.id.localeCompare(b.id));
+    }
+
+    return vote.slice().sort();
+  });
+}
+
+function expectVotesEqual(expected: VotesDict, actual: VotesDict) {
+  expect(sortVotes(actual)).toEqual(sortVotes(expected));
+}
+
+test('printing ballots', async () => {
```

Comment:
With the changes to the interpretation fixture flow, we can generate and interpret ballots dynamically, allowing us to catch more in backend testing than we previously could.